automata/agent/providers.py

```diff
@@ -165,7 +165,7 @@ def _build_initial_messages(
         assert "user_input_instructions" in instruction_formatter
 
         messages_config = load_config(
-            ConfigCategory.INSTRUCTION.value, self.config.instruction_version.value
+            ConfigCategory.INSTRUCTION.to_path(), self.config.instruction_version.to_path()
```

Comment:
How do we feel about the use of `to_path`? Maybe it would be better to override value or str. The to_path just converts kebab case to snake case.